<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>351</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>podatki</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btSave">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leUsername">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ime aplikacije</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leGeslo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Uporabni≈°ko ime</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leImeappa">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Geslo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leEmail">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>301</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Vrnjeni podatki</string>
   </property>
   <widget class="QLabel" name="lbVrnjeniPodatki">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>25</y>
      <width>271</width>
      <height>111</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Ni podatkov</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="leImeappa2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ime aplikacije</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btVrnigesla">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Vrni gesla</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>221</width>
     <height>491</height>
    </rect>
   </property>
   <property name="title">
    <string>Vsi podatki</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>221</width>
      <height>461</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
